nodeJs/processors/processMessage.js

```diff
@@ -2,15 +2,20 @@ import { mkdir, writeFile, stat } from "fs";
 import { formatDateForFileName } from "../helpers/dateHelpers.js";
 import path from "path";
 
-const DATA_DIRECTORY = "data";
+function isEmptyOrSpaces(string){
+  return string === null || string.match(/^\s*$/) !== null;
+}
 
-const args = process.argv.slice(2);
-const consoleOnly = args.includes("--console-only");
+const downloadDirectory = process.env.RELATIVE_FILE_DOWNLOAD_DIRECTORY
+
+if (isEmptyOrSpaces(downloadDirectory)) {
```

Comment:
Is this something we should do in `config.js`, perhaps?

Reply:
Move to `config.js`, thanks!